app/controllers/application_controller/ci_processing.rb

```diff
@@ -688,6 +688,7 @@ def generic_button_operation(action, action_name, operation, options = {})
   #           - false otherwise
   def testable_action(action)
     controller = params[:controller]
+    return false if action == 'destroy' && controller != 'host' # Change this if validate_destroy available also for other controllers
```

Comment:
Shouldn't this be written similarly with the rest of the routine?
```ruby
host_untestable_actions = %w(destroy)
...
if controller == 'host'
  return host_untestable_actions.exclude?(action)
end
...
```

Reply:
Hmm, you are right. I can do this. I was not sure about this and tried to fix it the simplest way. Thank you. I will update this PR asap.